Essentials/src/main/java/com/earth2me/essentials/commands/Commandice.java

```diff
@@ -0,0 +1,47 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.ChargeException;
+import com.earth2me.essentials.CommandSource;
+import com.earth2me.essentials.User;
+import com.earth2me.essentials.utils.VersionUtil;
+import net.ess3.api.IUser;
+import net.ess3.api.MaxMoneyException;
+import org.bukkit.Server;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commandice extends EssentialsLoopCommand {
```

Comment:
nit: maybe `freeze` instead of `ice` (to be more consistent in wording to other commands like `burn`)

Reply:
was talking with md about this in vc this morning and we decided that people may think that it would imply preventing the player from moving (like /freeze commands in other plugins do). Couldn't think of a better name other than ice.

Reply:
oh is that not what this does? i just saw setFreezeTicks and assumed they wouldn't be moving. point taken. 👍